src/Less/index.less

```diff
@@ -420,15 +425,14 @@ a[href=''] {
 	}
 }
 
-.wikiproject-selector[multiple] {
-  display: block;
-  height: 34px;
-  min-width: 330px; // Select2 doesn't know how to resize properly on smaller screens
+.searchbox {
+	float: none;
+	margin-top: 2px;
+	padding: 2px;
 }
 
-.wikiproject-label {
-  display: block;
-  margin-top: 10px;
+.search-label {
+  padding-right: 1%;
```

Comment:
Labels are inline elements, so you should set it to "display: inline-block" to avoid any weirdness. (Technically, you're not supposed to apply padding to inline elements.) Also, using a percentage here is a bit awkward. We use pixel-based margins and paddings for most of the other elements in the header.

Reply:
Done.